static/app/views/settings/projectMetrics/customMetricsTable.tsx

```diff
@@ -182,7 +184,7 @@ function MetricsTable({metrics, isLoading, query, project}: MetricsTableProps) {
                 </Tooltip>
               )}
               <Link
-                to={`/settings/projects/${project.slug}/metrics/${encodeURIComponent(
+                to={`/settings/${organization.slug}/projects/${project.slug}/metrics/${encodeURIComponent(
```

Comment:
I think you'll need `normalizeUrl()` on these string URLs to avoid a redirect in customer-domain environments.

Reply:
`Link` calls it https://github.com/getsentry/sentry/blob/master/static/app/components/links/link.tsx#L51